src/components/BookList.js

```diff
@@ -0,0 +1,28 @@
+import { PropTypes } from 'prop-types';
+import React from 'react';
+import BookItem from './BookItem';
+import AddBook from './AddBook';
+
+const BookList = ({ books, onDelete }) => (
+  <div className="book-list">
+    {books.map((book) => (
+      <BookItem key={book.id} book={book} onDelete={() => onDelete(book.id)} />
+    ))}
+    <AddBook />
+  </div>
+);
```

Comment:
- [ ] [OPTIONAL] I apologize for the confusion caused by the previous feedback. It's important to note that the `<BookList />` component and `<AddBook />` form component are structurally independent and are not intended to be nested within each other. Instead, I would recommend creating a parent component, such as `<Books>`, that can contain both the `<BookList />` and `<AddBook />` form components as its child components. This approach ensures a clear separation of concerns and follows recommended practices for component organization.

Reply:
Hello @AbelG101 , thank you for the pointer. The structure was not proper. 
Fixing it ASAP .

Cheers!